src/features/canvass/l10n/messageIds.ts

```diff
@@ -0,0 +1,138 @@
+import { ReactElement } from 'react';
+
+import { m, makeMessages } from 'core/i18n';
+
+export default makeMessages('feat.canvass', {
+  default: {
+    assignment: m('Untitled area assignment'),
```

Comment:
So far, we have used this as a fallback for assignments that have no title, and we do the same thing in multiple places (which means that this message exists in multiple places, right?).

This is not how it works for most other activities, e.g. call assignments, which must have a title. In those cases, we give it a default title when we create it. I think we should do that with area assignments too.